item.rb

```diff
@@ -0,0 +1,25 @@
+# item classs
+class Item
+  def initialize(date)
+    @id = Random.rand(1..1000)
+    @genre = nil
+    @author = nil
+    @label = nil
+    @publish_date = date
+    @archived = move_to_archive(@publish_date)
```

Comment:
Hello @saied2035 
- let make 
    ` attr_reader :genre, :label, :author
       attr_accessor :date`
and  

- let initialize @archived=false because move_to_archive should change the archived property to true if the result of the can_be_archived?() method is true. should do nothing if the result of the can_be_archived?() method is false.

- so in the method we can make 

 `def move_to_archive
    @archived = true if can_be_archived?
  end`

Reply:
Hello, @Abenezer-Tilahun. Thanks for the review. I will work on it.